website/versioned_docs/version-1.4.2/resources.md

```diff
@@ -30,6 +30,7 @@ original_id: resources
 ## Community Articles / Tutorials
 
 * [Building dynamic forms with Formik with React and TypeScript](https://scottdj92.ghost.io/building-dynamic-forms-with-formik-with-react-and-typescript/) by [Scott Jones](https://twitter.com/sdj2964) - Sep 1, 2018
+* [🍳 Render Props, Apollo and Formik: build and compose forms in React with ease 🏄](https://techblog.commercetools.com/render-props-apollo-and-formik-build-and-compose-forms-in-react-with-ease-f79a594be239) by [Tobias Deekens](https://github.com/tdeekens) - June 29, 2018
```

Comment:
This link 404s

Reply:
Sorry, it doesn't for me: https://techblog.commercetools.com/render-props-apollo-and-formik-build-and-compose-forms-in-react-with-ease-f79a594be239

Could you check again? I tried in three browsers all in inkognito mode.